var/spack/repos/builtin/packages/py-tweedledum/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyTweedledum(PythonPackage):
+    """tweedledum is a library for synthesis, compilation, and
+    optimization of quantum circuits. The library is written to be
+    scalable up to problem sizes in which quantum circuits outperform
+    classical ones. Also, it is meant to be used both independently
+    and alongside established tools."""
+
+    homepage = "https://github.com/boschmitt/tweedledum"
+    pypi = 'tweedledum/tweedledum-1.1.1.tar.gz'
+
+    version('1.1.1', sha256='58d6f7a988b10c31be3faa1faf3e58288ef7e8159584bfa6ded45742f390309f')
```

Comment:
Can you add a dep on Python 3.6+?

Reply:
Also missing a dep on `py-setuptools@42:`. Not sure which of these other deps are required:
```toml
[project]
requires-python = ">=3.6"

[build-system]
requires = [
    "setuptools>=42",
    "wheel",
    "cmake>=3.18",
    "ninja",
    "scikit-build>=0.12.0",
]
```